NodeJS/src/AgentClient.ts

```diff
@@ -250,9 +250,10 @@ export class AgentClient {
       slaPercentage: null,
       announcementExtension: null,
       redirectOnNoAnswerRings: null,
-      chatDigitalChannelsEnabled: null,
-      emailDigitalChannelsEnabled: null,
-      smsDigitalChannelsEnabled: null
+      chatDigitalChannelsEnabled: false,
+      emailDigitalChannelsEnabled: false,
+      smsDigitalChannelsEnabled: false,
+      dispositionSetId: ''
```

Comment:
Looks good. Is "clientId" parameter needed? I see that being sent from UI. One of these needs to be corrected :)
UI params:
UI:
{
    "clientId": 143,
    "name": "DEFAULT_SKILL",
    "skillNumber": 1974,
    "slaInSeconds": "",
    "slaPercentage": "",
    "acwInterval": "",
    "announcementExtension": "",
    "chatDigitalChannelsEnabled": false,
    "emailDigitalChannelsEnabled": false,
    "smsDigitalChannelsEnabled": false,
    "redirectOnNoAnswerRings": 3,
    "subAccountAppId": "SDK_SDKRec",
    "dispositionSetId": ""
}

Reply:
It is not needed.  We already have subAccountAppId in the request. UI should remove that.